lib/track.js

```diff
@@ -202,12 +211,14 @@ function windowDimensions() {
 Track.prototype.paste = function(e) {
   var sensitive = false;
   var panField = false;
-
   var props = {};
 
+  // Customisable logic for identifying pan and sensitive fields
+  var classifyPaste = this.classifyPaste(e);
+
   var t = e && e.target;
   if (t) {
-    panField = t.hasAttribute('data-rvn-pan');
+    panField = classifyPaste.pan;
```

Comment:
Let's keep the original behaviour and make the classifyPaste behaviour additional. Same for sensitive. Maybe:

```suggestion
    panField = t.hasAttribute('data-rvn-pan') || (classifyPaste && classifyPaste.pan);
```